browser/main/lib/AwsMobileAnalyticsConfig.js

```diff
@@ -3,7 +3,7 @@ const AMA = require('aws-sdk-mobile-analytics')
 const ConfigManager = require('browser/main/lib/ConfigManager')
 
 AWS.config.region = 'us-east-1'
-if (process.env.NODE_ENV === 'production') {
+if (process.env.NODE_ENV === 'production' && ConfigManager.default.get().amaEnabled) {
```

Comment:
This condition is necessary because unless it exists, the user, who just disable ama,  cannot disable sending data without reloading.